rust_rocks.md

```diff
@@ -0,0 +1,13 @@
+# Rust Rocks!
+
+Consider this the unofficial Rust sales pitch.
```

Comment:
The question is: shall we do it?

Reply:
I was unsure as well. And also didn't feel it's needed, but this file I feel the author wanted to use to introduce Rust to the general public as in `explain what it can do, it's limitations, borrowing, etc.`. But yeah, really not sure. :-/

Reply:
But maybe I would remove this file from the PR, factor out things that could be nice in other files as your stated with `Types` and then we discuss about a small introduction to Rust itself in another PR/discussion/issue?

Reply:
Removed rust_rocks.md and used introduction sentence in newtype.md